MetaMorpheus/GUI/TaskWindows/SearchTaskWindow.xaml

```diff
@@ -545,6 +545,27 @@
                                     <ToolTip Content="Use peptide sequences shared between proteins for protein quantification" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="500" />
                                 </ToolTipService.ToolTip>
                             </CheckBox>
+                            <StackPanel Orientation="Horizontal">
+                                <StackPanel.IsEnabled>
+                                    <MultiBinding Converter="{StaticResource boolOrConverter}">
+                                        <Binding  ElementName="CheckBoxLFQ" Path ="IsChecked"/>
+                                        <Binding  ElementName="CheckBoxLFQwSpectralRecovery" Path ="IsChecked"/>
+                                    </MultiBinding>
+                                </StackPanel.IsEnabled>
+                                <StackPanel.Visibility>
+                                    <MultiBinding Converter="{StaticResource boolOr2VisConverter}">
+                                        <Binding  ElementName="CheckBoxLFQ" Path ="IsChecked"/>
+                                        <Binding  ElementName="CheckBoxLFQwSpectralRecovery" Path ="IsChecked"/>
+                                    </MultiBinding>
+                                </StackPanel.Visibility>
+                                
+                                <local:DoubleTextBoxControl x:Name="MbrFdrThresholdTextBox" Width="45" Height="20">
+                                    <ToolTipService.ToolTip>
+                                        <ToolTip Content="FDR Threshold for MBR Results" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="500" />
```

Comment:
Should say that this is the q-value cut-off to select peptides used for label free quant. Could say that there may be some PSMs with q-value below cut-off that from peptides at greater than q-value cut-off and these will not be quantified. Also, all psms with q-value below cut-off will be used to mark a peptide as MSMS identified

Reply:
> Should say that this is the q-value cut-off to select peptides used for label free quant.
That's not what it does. This controls the q-value for PIP results. I've updated the text to make this more clear